index.js

```diff
@@ -451,47 +471,33 @@ app.post('/user/follower',(req,res)=>{
   });
 });
 
-app.post('/user/chat',(req, res) => {
-  const {uid} = req.body;
-  var qry = `SELECT * FROM USERS WHERE login = '${uid}'`
-  pool.query(qry,(err,result)=>{
-    if(err)
-      res.send(err);
-    res.render('pages/chat', {user:req.user,pageTitle:"Chat",path:'/user/chat',user:req.user});
-  });
+app.get('/chat/:name',checkNotAuthenticated,(req,res)=>{
+  var name = req.params.name
+    var query = `select * from Users where login = $1`;
+    pool.query(query,[name],(error,result)=>{
+      if(error)
+        res.send(error);
+      var results = {'attributes':result.rows[0]};
+      res.render('pages/chat',{row:results,pageTitle: 'Grababite • Chat', path: '/chat'+`${name}`, user: req.user})
+    });
 });
 
-io.sockets.on('connection', (socket) => {
-  // message
-  var roomName = null;
+const users = {}
+io.sockets.on('connection', socket =>
```

Comment:
the index.js does not even run because there is syntax error you forgot a '{' to start the function definition. Should be socket => {